xlive/Blam/Engine/cseries/cseries.h

```diff
@@ -210,7 +212,7 @@ int32 vsprintf(char* buffer, size_t size, const char* format, va_list va_args);
 * NOTES:
 * size_t max_count was added in vista to this function call
 */
-int32 vsnprintf(char* buffer, size_t size, size_t max_count, const char* format, va_list va_args);
+int32 vsnprintf(char* buffer, size_t size, size_t max_count, const char* format, char* ap);
```

Comment:
vsnprintf is a library function? 

https://cplusplus.com/reference/cstdio/vsnprintf/

Reply:
Bungie would end up re-using the same CRT function names for their cseries wrappers for halo 2.
This is intentional.

Reply:
then why did you change the parameter?